base_layer/core/src/chain_storage/lmdb_db/lmdb.rs

```diff
@@ -66,10 +66,11 @@ where
     let val_buf = serialize(val, None)?;
     match txn.access().put(db, key, &val_buf, put::NOOVERWRITE) {
         Ok(_) => {
-            trace!(
-                target: LOG_TARGET, "Inserted {} bytes with key '{}' into '{}'",
-                val_buf.len(), to_hex(key.as_lmdb_bytes()), table_name
-            );
+            // Note: Enable when detail trace logs are required for testing - this fills up trace logs pretty quickly
+            // trace!(
+            //     target: LOG_TARGET, "Inserted {} bytes with key '{}' into '{}'",
+            //     val_buf.len(), to_hex(key.as_lmdb_bytes()), table_name
+            // );
```

Comment:
```suggestion
             trace!(
                 target: LOG_TARGET_X, "Inserted {} bytes with key '{}' into '{}'",
                 val_buf.len(), to_hex(key.as_lmdb_bytes()), table_name
             );
```
Either delete this, keep as is or change the target to something wth a postfix, I prefer the later, but does not matter much